Alignment/OfflineValidation/plugins/BuildFile.xml

```diff
@@ -5,6 +5,7 @@
 <use   name="CondCore/DBOutputService"/>
 <use   name="CondFormats/Alignment"/>
 <use   name="CondFormats/RunInfo"/>
+<use   name="CondFormats/SiPixelObjects"/>
```

Comment:
same comment here

Reply:
It is used here: https://github.com/cms-sw/cmssw/blob/master/Alignment/OfflineValidation/plugins/TrackerGeometryCompare.cc#L18

Again, this was not an issue before because it somehow got this dependency indirectly via some RecoTracker package, but now it's done right.